grafana/RELEASES.md

```diff
@@ -1,5 +1,8 @@
 # Release Notes
 
+## 0.7 (Unreleased)
```

Comment:
Note that it says unreleased. I am just tracking all the changes as we make
them under a new release label.

On Mon, Apr 13, 2015 at 5:39 PM, Victor Marmol <notifications@github.com>
wrote:

> In grafana/RELEASES.md
> <https://github.com/GoogleCloudPlatform/heapster/pull/236#discussion_r28292850>
> :
>
> > @@ -1,5 +1,8 @@
> >  # Release Notes
> >
> > +## 0.7 (Unreleased)
>
> Was this meant to be here? Maybe do in another PR or commit? We should
> probably mention some of the GCM changes as well.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/heapster/pull/236/files#r28292850>
> .
>